collector/netframework_clrsecurity.go

```diff
@@ -0,0 +1,118 @@
+// returns data points from Win32_PerfRawData_NETFramework_NETCLRSecurity
+// <add link to documentation here> - Win32_PerfRawData_NETFramework_NETCLRSecurity class
+package collector
+
+import (
+	"log"
+
+	"github.com/StackExchange/wmi"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func init() {
+	Factories["netframework_clrsecurity"] = NewNETFramework_NETCLRSecurityCollector
+}
+
+// A NETFramework_NETCLRSecurityCollector is a Prometheus collector for WMI Win32_PerfRawData_NETFramework_NETCLRSecurity metrics
+type NETFramework_NETCLRSecurityCollector struct {
+	NumberLinkTimeChecks *prometheus.Desc
+	TimeinRTchecks       *prometheus.Desc
+	StackWalkDepth       *prometheus.Desc
+	TotalRuntimeChecks   *prometheus.Desc
+}
+
+// NewNETFramework_NETCLRSecurityCollector ...
+func NewNETFramework_NETCLRSecurityCollector() (Collector, error) {
+	const subsystem = "netframework_clrsecurity"
+	return &NETFramework_NETCLRSecurityCollector{
+		NumberLinkTimeChecks: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "link_time_checks_total"),
+			"Displays the total number of link-time code access security checks since the application started.",
+			[]string{"process"},
+			nil,
+		),
+		TimeinRTchecks: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "rt_checks_timein"),
```

Comment:
`rt_checks_time_seconds_total`

Reply:
Similarly to the the others I need to double check the meaning.